.github/workflows/cleanup.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   cleanup-non-db:
     name: Delete non-DB resources associated with pr
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     steps:
       - name: Set Variables
         shell: bash
```

Comment:
Might as well bump up the Ubuntu version on the `cleanup-db` job in this file as well.

Reply:
Done in: https://github.com/bcgov/wps/pull/3995/commits/dfaba153d40a319d5f8f57cd8c2df4b83ea055ef